mobile_verifier/src/heartbeats.rs

```diff
@@ -158,45 +212,47 @@ pub struct Heartbeat {
     pub cell_type: Option<CellType>,
```

Comment:
Similar to the comment on speedtests.rs, this is duplicating an existing filestore struct `CellHeartbeat` and adding a validity field